package.json

```diff
@@ -89,7 +85,7 @@
     "pre_test": "npm run lint",
     "tape": "node test/tape/index.js | tap-spec",
     "debug_tape": "node --inspect-brk test/tape/index.js",
-    "test": "npm run pre_test && nyc mocha --exit --check-leaks -t 10000 -R spec test/index.js && npm run tape",
+    "test": "npm run pre_test && nyc mocha --exit --check-leaks --globals __core-js_shared__ -t 10000 -R spec test/index.js && npm run tape",
```

Comment:
Ignore for "__core-js_shared__" was added since currently there is no way around it, Tedious brings babel-runtime into test execution and that results in an expected leakage (see https://github.com/babel/babel-loader/issues/152). This is not a problem with Knex nor something we should address.